main.tf

```diff
@@ -65,104 +84,31 @@ resource "google_compute_instance" "elasticsearch" {
 
   network_interface {
     network = var.network
-
-    access_config {
-      // Ephemeral IP
-    }
   }
 
   metadata = {
-    ssh-keys = "devops:${tls_private_key.provision.public_key_openssh}"
+    ssh-keys  = "devops:${tls_private_key.provision.public_key_openssh}"
+    user-data = <<-EOT
+#!/bin/bash
+
+echo "${base64encode(local.elasticsearch_configuration)}" | base64 -d > /tmp/elasticsearch.yml;
+echo "${google_service_account_key.elasticsearch_backup.private_key}" | base64 -d > /tmp/backup-sa.key;
+echo "${base64encode(local.master_list)}" | base64 -d > /home/devops/master_list.sh;
+echo "${filebase64("${path.module}/bootstrap.sh")}" | base64 -d > /tmp/bootstrap.sh;
```

Comment:
The same as the above.

Reply:
https://github.com/AckeeCZ/terraform-gcp-elasticsearch/pull/3#discussion_r459914335 :trollface: